compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/ir/IrUtils.kt

```diff
@@ -96,6 +99,12 @@ fun IrType.getArrayElementType(irBuiltIns: IrBuiltIns): IrType =
     else
         irBuiltIns.primitiveArrayElementTypes.getValue(this.classOrNull!!)
 
+val IrStatementOrigin?.isLambda
```

Comment:
Would make sense to delete similar utility from `CallableReferenceLowering`

Also please add explicit return type

Reply:
Also, this function can be used in `IrBlock.isInlineIrBlock` now

Reply:
I must have missed the one in `CallableReferenceLowering`, thanks for pointing it out! Also changed `isInlineIrBlock` to use this.